### PR TITLE
Update teams-powershell-install.md

### PR DESCRIPTION
**Pull request by pupara**
Updated content

### CODE REVIEW

**Comment by Kateyanne** [comment]
@JohanFreelancer9 please copyedit. @scanum fyi

File: Teams/teams-powershell-install.md [hunk 1]
```diff
@@ -15,32 +15,22 @@ appliesto:
...
+# Install Microsoft Teams PowerShell Module
 
-This article explains how to install the Microsoft Teams PowerShell module using
-[PowerShellGet](/powershell/scripting/gallery/installing-psget). These instructions work on [Azure Cloud Shell](/azure/cloud-shell/overview), Linux, macOS, and Windows platforms.
+This article explains how to install the Microsoft Teams PowerShell module using PowerShell Gallery. The Microsoft Teams PowerShell module is supported on all Windows platforms. 
 
 ## Requirements
 
-Teams PowerShell requires PowerShell 5.1 or higher on all platforms. Install the [latest version of PowerShell](/powershell/scripting/install/installing-powershell) available for your operating system.
+Microsoft Teams PowerShell module requires PowerShell 5.1 or higher on all platforms. Install the [latest version of PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell) available for your operating system.
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:24]:
```suggestion
Microsoft Teams PowerShell module requires PowerShell 5.1 or higher on all platforms. Install the [latest version of PowerShell](/powershell/scripting/install/installing-powershell) available for your operating system. 
```

File: Teams/teams-powershell-install.md [hunk 2]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
-## Sign in
-
-To start working with Teams PowerShell, sign in with your Azure credentials.
+## Installing using the PowerShellGallery
 
-> [!NOTE]
-> If you're using the latest [Teams PowerShell public preview release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
+Microsoft Teams PowerShell module is currently supported for use with PowerShell 5.1 on Windows. Plase follow below steps to install the module: 
 
+- Update to [Windows PowerShell 5.1](https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/install/installing-windows-powershell#upgrading-existing-windows-powershell). If you're on Windows 10 version 1607 or higher, you already have PowerShell 5.1 installed.
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:51]:
```suggestion
- Update to [Windows PowerShell 5.1](/powershell/scripting/windows-powershell/install/installing-windows-powershell#upgrading-existing-windows-powershell). If you're on Windows 10 version 1607 or higher, you already have PowerShell 5.1 installed. 
```

File: Teams/teams-powershell-install.md [hunk 3]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
-
-To start working with Teams PowerShell, sign in with your Azure credentials.
+## Installing using the PowerShellGallery
 
-> [!NOTE]
-> If you're using the latest [Teams PowerShell public preview release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
+Microsoft Teams PowerShell module is currently supported for use with PowerShell 5.1 on Windows. Plase follow below steps to install the module: 
 
+- Update to [Windows PowerShell 5.1](https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/install/installing-windows-powershell#upgrading-existing-windows-powershell). If you're on Windows 10 version 1607 or higher, you already have PowerShell 5.1 installed. 
+- Install [.NET Framework 4.7.2](https://docs.microsoft.com/en-us/dotnet/framework/install) or later.
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:52]:
```suggestion
- Install [.NET Framework 4.7.2](/dotnet/framework/install) or later. 
```

File: Teams/teams-powershell-install.md [hunk 4]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
+
+```PowerShell
+Connect-MicrosoftTeams 
+``` 
 
-## Update Teams PowerShell
+## Update Teams PowerShell Module
 
-To update Teams PowerShell, open a new elevated PowerShell command prompt and run the following:
+To update any PowerShell module, you should use the same method used to install the module. For example, if you originally used Install-Module, then you should use [Update-Module](https://docs.microsoft.com/en-us/powershell/module/powershellget/update-module) to get the latest version.
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:78]:
```suggestion
To update any PowerShell module, you should use the same method used to install the module. For example, if you originally used Install-Module, then you should use [Update-Module](/powershell/module/powershellget/update-module) to get the latest version.  
```

File: Teams/teams-powershell-install.md [hunk 5]
```diff
@@ -91,40 +87,18 @@ Update-Module MicrosoftTeams
...
-Install-Module MicrosoftTeams -AllowPrerelease -RequiredVersion "1.1.9-preview"
+# Delete all version of the module
+Uninstall-Module MicrosoftTeams -Allversion 
 ```
 
-## Next Steps
+# Next Steps 
 
-Now you're ready to manage Teams using Teams PowerShell. See [Managing Teams with Teams PowerShell](teams-powershell-managing-teams.md) to get started.
+Now you're ready to manage Microsoft Teams using Microsoft Teams PowerShell. See [Managing Teams with Teams PowerShell](https://docs.microsoft.com/en-us/microsoftteams/teams-powershell-managing-teams) to get started.
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:101]:
```suggestion
Now you're ready to manage Microsoft Teams using Microsoft Teams PowerShell. See [Managing Teams with Teams PowerShell](teams-powershell-managing-teams.md) to get started. 
```

File: Teams/teams-powershell-install.md [hunk 6]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
- If your account uses multi-factor authentication, use the steps in this section.
+In some environments, it's not possible to connect to the PowerShell Gallery. In those situations, please follow these [manual installation steps](https://aka.ms/psgallery-manualdownload).  
 
-```powershell
-#Connect to Microsoft Teams
-Connect-MicrosoftTeams -AccountId <UPN>
-```
+## Sign in
+
+To start working with Teams PowerShell, sign in with your Azure credentials.
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:70]:
```suggestion
To start working with Microsoft Teams PowerShell module, sign in with your Azure credentials.
```

File: Teams/teams-powershell-install.md [hunk 7]
```diff
@@ -91,40 +87,18 @@ Update-Module MicrosoftTeams
...
-```
-
-To install Teams PowerShell public preview, run the PowerShell command below.
-
-> [!NOTE]
-> You can find the latest preview version at [PowerShell Gallery](https://www.powershellgallery.com/packages/MicrosoftTeams) or in PowerShell by running "Find-Module MicrosoftTeams -AllowPrerelease -AllVersions"
-
-```powershell
-Install-Module MicrosoftTeams -AllowPrerelease -RequiredVersion "1.1.9-preview"
+# Delete all version of the module
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:95]:
```suggestion
# Uninstall all versions of the module
```

File: Teams/teams-powershell-install.md [hunk 8]
```diff
@@ -91,40 +87,18 @@ Update-Module MicrosoftTeams
...
+Uninstall-Module MicrosoftTeams -Allversion
```
Comment (get-itips) [inline Teams/teams-powershell-install.md:96]:
```suggestion
Uninstall-Module MicrosoftTeams -Allversions 
```

File: Teams/teams-powershell-install.md [hunk 9]
```diff
@@ -15,32 +15,22 @@ appliesto:
...
 
 ```powershell
-Install-Module MicrosoftTeams
+$PSVersionTable.PSVersion 
 ```
-
-By default, the PowerShell Gallery (PSGallery) isn't configured as a trusted repository for **PowerShellGet**. The first time you use the PSGallery, you'll see the following message:
+It is recommended that you use the  Install-Module cmdlet to install the Microsoft Teams PowerShell module. 
+ 
+If PowerShell Gallery (PSGallery) isn't configured as a trusted repository for **PowerShellGet**. The first time you use the PSGallery, you'll see the following message:
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:33]:
```suggestion
If PowerShell Gallery (PSGallery) isn't configured as a trusted repository for **PowerShellGet**. The first time you use the PSGallery, you'll see the following message:
```
```suggestion
If PowerShell Gallery (PSGallery) isn't configured as a trusted repository for **PowerShellGet**. The first time you use the PSGallery, you'll see the following message:
```
```suggestion
If PowerShell Gallery (PSGallery) isn't configured as a trusted repository for **PowerShellGet**, the first time you use the PSGallery you'll see the following message:
```

File: Teams/teams-powershell-install.md [hunk 10]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
 Answer **Yes** or **Yes to All** to continue with the installation.
 
-## Sign in
-
-To start working with Teams PowerShell, sign in with your Azure credentials.
+## Installing using the PowerShellGallery
 
-> [!NOTE]
-> If you're using the latest [Teams PowerShell public preview release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
+Microsoft Teams PowerShell module is currently supported for use with PowerShell 5.1 on Windows. Plase follow below steps to install the module:
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:49]:
```suggestion
Microsoft Teams PowerShell module is currently supported for use with PowerShell 5.1 on Windows. Please follow the following steps to install the module: 
```

File: Teams/teams-powershell-install.md [hunk 11]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
-To start working with Teams PowerShell, sign in with your Azure credentials.
+## Installing using the PowerShellGallery
 
-> [!NOTE]
-> If you're using the latest [Teams PowerShell public preview release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
+Microsoft Teams PowerShell module is currently supported for use with PowerShell 5.1 on Windows. Plase follow below steps to install the module: 
 
+- Update to [Windows PowerShell 5.1](/powershell/scripting/windows-powershell/install/installing-windows-powershell#upgrading-existing-windows-powershell). If you're on Windows 10 version 1607 or higher, you already have PowerShell 5.1 installed. 
+- Install [.NET Framework 4.7.2](/dotnet/framework/install) or later. 
+- Run the following command to install latest PowerShellGet:
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:53]:
```suggestion
- Run the following command to install the latest PowerShellGet:
```

File: Teams/teams-powershell-install.md [hunk 12]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
+ 
 ```powershell
-$credential = Get-Credential
+Install-Module -Name PowerShellGet -Force -AllowClobber
+```
+- Install the Teams PowerShell Module
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:58]:
```suggestion
- Install the Teams PowerShell Module.
```

File: Teams/teams-powershell-install.md [hunk 13]
```diff
@@ -54,32 +44,38 @@ Are you sure you want to install the modules from 'PSGallery'?
...
 
-## Update Teams PowerShell
+```PowerShell
+Connect-MicrosoftTeams 
+``` 
 
-To update Teams PowerShell, open a new elevated PowerShell command prompt and run the following:
+## Update Teams PowerShell Module
+
+To update any PowerShell module, you should use the same method used to install the module. For example, if you originally used Install-Module, then you should use [Update-Module](https://docs.microsoft.com/en-us/powershell/module/powershellget/update-module) to get the latest version.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:78]:
```suggestion
To update any PowerShell module, you should use the same method used to install the module. For example, if you originally used Install-Module, then you should use [Update-Module](/powershell/module/powershellget/update-module) to get the latest version.  
```

Comment (JohanFreelancer9) [reply Teams/teams-powershell-install.md:78]: Please change the link to a relative link.

File: Teams/teams-powershell-install.md [hunk 14]
```diff
@@ -15,32 +15,22 @@ appliesto:
...
-session using **Run as administrator** in Windows or use the `sudo` command on macOS or Linux:
+To check your PowerShell version, run the following command from within a PowerShell session: 
 
 ```powershell
-Install-Module MicrosoftTeams
+$PSVersionTable.PSVersion 
 ```
-
-By default, the PowerShell Gallery (PSGallery) isn't configured as a trusted repository for **PowerShellGet**. The first time you use the PSGallery, you'll see the following message:
+It is recommended that you use the  Install-Module cmdlet to install the Microsoft Teams PowerShell module.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:31]:
```suggestion
We recommend that you use the  Install-Module cmdlet to install the Microsoft Teams PowerShell module. 
```

File: Teams/teams-powershell-install.md [hunk 15]
```diff
@@ -91,40 +87,18 @@ Update-Module MicrosoftTeams
...
-> You can find the latest preview version at [PowerShell Gallery](https://www.powershellgallery.com/packages/MicrosoftTeams) or in PowerShell by running "Find-Module MicrosoftTeams -AllowPrerelease -AllVersions"
-
-```powershell
-Install-Module MicrosoftTeams -AllowPrerelease -RequiredVersion "1.1.9-preview"
+# Uninstall all versions of the module
+Uninstall-Module MicrosoftTeams -Allversions 
 ```
 
-## Next Steps
+# Next Steps
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-install.md:99]:
```suggestion
## Next Steps 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pupara, Please commit my suggested changes. Many thanks.

**Review by dariomws — APPROVED** [review]

**Pull request merged**